.gitignore

```diff
@@ -1,3 +1,3 @@
 unpacked-extension
 tako-github-file-tree.zip
-node_modules
\ No newline at end of file
+node_modules
```

Comment:
Thanks! FYI: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
That's interesting but I don't think it's something you should depend on in open source projects.